cf-oscore/src/main/java/org/eclipse/californium/oscore/HashMapCtxDB.java

```diff
@@ -116,7 +109,9 @@ public synchronized void addContext(Token token, OSCoreCtx ctx) {
 	@Override
 	public synchronized void addContext(String uri, OSCoreCtx ctx) throws OSException {
 		if (uri != null) {
-			uriMap.put(normalizeServerUri(uri), ctx);
+		    String normalizedUri = normalizeServerUri(uri);
```

Comment:
My editor shows
\t\t\s\s\s\s
should be
\t\t\t

Do you like to reformat it? Or should we just merge it, "as it is"?

Reply:
Fixed in commit 6b98ac703fd398530455b8c5ea3f9e617268fed0